### DISCUSSION TITLE
[UI] Streamline workflow with "Paste from Clipboard"

### DISCUSSION THREAD

**Discussion by codedealer** [hlky/stable-diffusion-webui#435] [Feature suggestions]
I am sure we're all tired from countless "Push x to y" buttons under every `Gallery` output, there are so many of them yet still not enough.

**Proposal**:
- [ ] Remove all the "Push..." buttons
- [ ] Each `Gallery output` gets "Copy to Clipboard"
- [ ] Each `Image` input gets "Paste from Clipboard"

This will declutter the UI and as a sweet bonus opens a (almost) seamless integration with your favorite tools! Paste directly from Photoshop/Krita/Google Images, move images from img2img to ESRGAN and back — all with to simple buttons.

**Relevant context**
Currently there is an open issue on Gradio gradio-app/gradio#735 to implement paste from clipboard but it's been open for half a year and seems pretty low on their priority list.
So, I figured we manage ourselves for the time being?
@altryne shameless tag since you're spearheading the UI overhaul currently

**Comment by altryne** [comment]
Saw this in the morning and had some time to mull this over. 
Agree that the buttons are... getting out of hand. 
From the other side, I really don't want to "simplify" the interface by adding 3 clicks for the same function folks use now. 
Push changes the tab, and loads the image, without going through the users clipboard. 

Paste from clipboard is a great suggestion! I absolutely think we can add this while we wait for Gradio. 
And I cleaned up the button area a little bit in #421 , and .. we should somehow toe the balance of functionality and UI coherence. 

Thanks for this, let's keep the discussion
